qa/integration-tests/src/test/java/io/camunda/it/auth/AuthorizationSearchIT.java

```diff
@@ -47,28 +51,18 @@ class AuthorizationSearchIT {
           ADMIN,
           DEFAULT_PASSWORD,
           List.of(new Permissions(AUTHORIZATION, PermissionTypeEnum.READ, List.of("*"))));
-
   private static final User RESTRICTED_USER = new User(RESTRICTED, DEFAULT_PASSWORD, List.of());
```

Comment:
🔧 I'm thinking about introducing a marker annotation, so to make more clear that the extension is actually consuming these properties, to create permissions,etc. and for the authenticated clients.

Reply:
We should do that, otherwise it's just unclear that these are picked up by the extension 😅

Reply:
@megglos I introduced with https://github.com/camunda/camunda/pull/28746/commits/ea650fc97098fb54e962e7f7fd792e39eca2a7e0 a new marker interface.

Locally all tests succeed.

I also extended the docs to make this clear. https://github.com/camunda/camunda/pull/28746/commits/1da4fb7f2bea64f86544411f8fb37b3fe08c0916